src/lib/__tests__/InstantSearch-test.js

```diff
@@ -516,6 +516,21 @@ describe('InstantSearch lifecycle', () => {
     expect(helperSearchSpy).toHaveBeenCalledTimes(1);
     expect(search.started).toBe(false);
   });
+
+  it('should set the helper to `null`', () => {
```

Comment:
Why is this test in the test suite `"When adding widgets after start"`?

Reply:
I copied it and didn't realise it was in the wrong spot